src/write_gcm.c

```diff
@@ -509,7 +509,7 @@ static EVP_PKEY *wg_credential_contex_load_pkey(char const *filename,
 // Otherwise returns 0.
 static int wg_curl_get_or_post(char *response_buffer,
     size_t response_buffer_size, const char *url, const char *body,
-    const char **headers, int num_headers);
+    const char **headers, int num_headers, _Bool expect_failure);
```

Comment:
Now that I look at it, `expect_failure` seems to imply that we *want* this to fail. Maybe `ok_to_fail` would sound better? Or we could go back to `ignore_failure`... @qingling128, since you proposed the name, WDYT?

Reply:
That makes sense. I just thought `ignore_failure` sounds a bit passive. `ok_to_fail` sounds good.

Reply:
Just thought of something even better: `silent_failures`. This reflects the exact behavior and is not passive.